packages/@react-aria/dnd/src/useDrag.ts

```diff
@@ -332,6 +336,18 @@ export function useDrag(options: DragOptions): DragResult {
     };
   }
 
+  if (isDisabled) {
+    return {
+      dragProps: {
+        draggable: 'false'
+      },
+      dragButtonProps: {
+        onPress() {}
```

Comment:
just curious, was this required? Any reason we shouldn't just provide an empty obj?

Reply:
No reason, just updated that